Mathlib/Algebra/Lie/Subalgebra.lean

```diff
@@ -107,6 +107,9 @@ instance [SMul R₁ R] [Module R₁ L] [IsScalarTower R₁ R L] (L' : LieSubalge
 instance (L' : LieSubalgebra R L) [IsNoetherian R L] : IsNoetherian R L' :=
   isNoetherian_submodule' _
 
+instance (L' : LieSubalgebra R L) [IsArtinian R L] : IsArtinian R L' :=
```

Comment:
This could probably be moved downstream to avoid additional imports.

Reply:
Agreed but I'm fine with the import bump if you are.